client/src/hooks/helpers/useArmies.tsx

```diff
@@ -73,57 +73,41 @@ const formatArmies = (
       }
       if (!owner) return undefined;
 
-      let healthClone = structuredClone(health);
-      if (healthClone) {
-        healthClone.current =
-          healthClone.current /
+      if (health) {
+        health.current =
+          health.current /
           (BigInt(EternumGlobalConfig.resources.resourcePrecision) * EternumGlobalConfig.troop.healthPrecision);
-        healthClone.lifetime =
-          healthClone.lifetime /
+        health.lifetime =
+          health.lifetime /
           (BigInt(EternumGlobalConfig.resources.resourcePrecision) * EternumGlobalConfig.troop.healthPrecision);
       } else {
-        healthClone = {
+        health = {
           entity_id: army.entity_id,
           current: 0n,
           lifetime: 0n,
         };
       }
       const protectee = getComponentValue(Protectee, armyEntityId);
 
-      const quantity = getComponentValue(Quantity, armyEntityId);
-      let quantityClone = structuredClone(quantity);
-      if (quantityClone) {
-        quantityClone.value = BigInt(quantityClone.value) / BigInt(EternumGlobalConfig.resources.resourcePrecision);
+      let quantity = structuredClone(getComponentValue(Quantity, armyEntityId));
+      if (quantity) {
+        quantity.value = BigInt(quantity.value) / BigInt(EternumGlobalConfig.resources.resourcePrecision);
       } else {
-        quantityClone = {
+        quantity = {
           entity_id: army.entity_id,
           value: 0n,
         };
       }
 
       const movable = getComponentValue(Movable, armyEntityId);
 
-      const capacity = getComponentValue(Capacity, armyEntityId);
-      let capacityClone = structuredClone(capacity);
-      if (capacityClone) {
-        capacityClone.weight_gram = capacityClone.weight_gram * quantityClone.value;
-      } else {
-        capacityClone = {
-          entity_id: army.entity_id,
-          weight_gram: BigInt(EternumGlobalConfig.carryCapacity.army) * quantityClone.value,
-        };
-      }
+      const capacity = structuredClone(getComponentValue(Capacity, armyEntityId));
```

Comment:
is there a need for structuredClone here ?

Reply:
removed it, my bad